pkg/reconciler/inmemorychannel/dispatcher/inmemorychannel_test.go

```diff
@@ -408,6 +499,7 @@ func TestReconciler_ReconcileKind(t *testing.T) {
 		},
 	}
 	for n, tc := range testCases {
+		ctx, fakeEventingClient := fakeeventingclient.With(context.Background(), tc.imc)
```

Comment:
That's the "indirect" injection I was talking about in the previous comment, just to clarify.